benchmarks/static_runtime/deep_wide_pt.cc

```diff
@@ -38,6 +38,16 @@ const std::string trivial_model_1 = R"JIT(
       return a + b * c + s
 )JIT";
 
+const std::string leaky_relu_model = R"JIT(
```

Comment:
looks great! can we also get a benchmark in that tests a non-constant value for leaky_relu?  I'm very interested in the overhead of `toScalar` in this setting